### PR TITLE
OSL-423: removing xss library, switching to using entities library for HTML encoding

### PR DESCRIPTION
**Pull request by katerinachinnappan**
## Goal

`xss` package doesn't seem to be consistent, so switching to `entities` package.

## Tickets

- More details here: [https://getpocket.atlassian.net/browse/OSL-423](https://getpocket.atlassian.net/browse/OSL-423)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -37,6 +37,7 @@
     "aws-xray-sdk-core": "3.4.1",
     "aws-xray-sdk-express": "3.4.1",
     "cors": "2.8.5",
+    "entities": "^4.5.0",
```
Comment (kschelonka) [inline package.json:40]:
```suggestion
    "entities": "4.5.0",
```
so renovate can manage

Comment (nina-py) [reply package.json:40]: I think Renovate bot will pin a new package automatically to a patch version when there's an update, so this could be left as-is

**Review by nina-py — APPROVED** [review]
Great choice

**Comment by katerinachinnappan** [comment]
closing for now, keeping branch alive

**Pull request closed without merging**